engine/robot_logic/traversal.py

```diff
@@ -208,64 +212,98 @@ def update_state(robot_state, velocity, omega):
     return measurements
 
 
-def traverse_roomba(robot_state, base_station_loc, time_limit, roomba_radius):
+def traverse_roomba(robot_state, base_station_loc, time_limit, roomba_radius, database):
     """Move the robot in a roomba-like manner.
     Args:
         base_station_loc (Tuple): The (x,y) location of the base station
         time_limit (Double): How long the robot will move using roomba mode
         roomba_radius (Double): Maximum radius from the base station that the robot can move
     Returns:
-        None
+        robot_state
     """
     # to add obstacle avoidance, can either change roomba to go to point on the circle in the direction of the
     # robot direction or use raw sensor value
     # can we guarantee that when giving a velocity that the robot will move at that velocity?
-    dt = 0
+    dt = 10
```

Comment:
why is dt = 10?

Reply:
could be any value. its just what we're passing to time.sleep(). smaller value means update faster but also hog cpu more. i think 10 is usually standard. at least we've always been using time.sleep(10) in our while loops

Reply:
oh yeah I just saw later on dt was also used in time.sleep(). That makes sense!